### PR TITLE
Kp todo

### LINKED ISSUE #16: KP_TODO
Spots that @kaizadp needs to look at are marked with `KP_TODO`.

### PR DESCRIPTION
**Pull request by kaizadp**
- fixed KP_TODO:
  - function descriptions
  - Hertkorn2013 data description/rda
  - @bpbond Does this look ok?
- rename `gg_spectra` -> `nor_plot_spectra`
- updated function descriptions (.Rd files)
- added authors

Closes #16

### CODE REVIEW

File: R/1-functions_graphs.R
Comment (bpbond) [inline R/1-functions_graphs.R:23]: Provide an `@examples`?

File: R/1-functions_processing.R
Comment (bpbond) [inline R/1-functions_processing.R]: "Software"

File: R/data.R
Comment (bpbond) [inline R/data.R:10]: Beautiful—thank you!

**Review by bpbond — CHANGES_REQUESTED** [review]
Looks good! A few minor suggestions below.

**Comment by kaizadp** [comment]
Thanks, changes made!

fbd85503a1c312d083f5a3f4ed88c2acbf8b0c9a 
237a05761c66273cb219503b6b950a67e99e3f35

The `nmr_plot_spectra` example is a little weird for meb_burn data, because the intensities are very high and have not been normalized. Will update the example later, when Morgan gives me simpler, normalized data.

**Comment by bpbond** [comment]
So as a general rule of thumb: please don't merge PRs with failing tests. I will look at this later.

**Comment by kaizadp** [comment]
Good point! I think I know what’s causing the tests to fail, so I’m working on it right now.
Next time, I’ll fix before merging. 
Thanks!

**Comment by bpbond** [comment]
Thanks! Sorry if that sounded grumpy :)

**Comment by kaizadp** [comment]
No worries, I was impatient to merge branches.

PS: there's still an error. The previous error was because I changed the function name (`nmr_plot_spectra`) but didn't update the tests and documentation. I did so, and now I get this error:
(not urgent, when you get a chance later)

image

**Pull request merged**
